jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -1,5 +1,6 @@
 import os
 import pickle
+from time import sleep
```

Comment:
```suggestion
```

Reply:
Whoops, thought I removed that, good catch